dashboard/src/components/LoadingWrapper/LoadingPlaceholder/LoadingPlaceholder.scss

```diff
@@ -0,0 +1,25 @@
+.LoadingPlaceholder {
+  height: 1em;
+  background-color: #f1f1f1;
```

Comment:
Are you sure that this is the minimum amount of CSS that we need? Do we need the background color set here?

Reply:
yes, this is the grey background colour that the lighter one animates over